RELEASING.md

```diff
@@ -40,3 +40,33 @@ You should have no local changes and tests should be passing.
 1. Close the milestone for this release. https://github.com/tigera/operator/milestones
 
 1. Go to https://github.com/tigera/operator/milestones and create any new milestones that should exist (e.g., next patch release)
+
+## Creating a release on RH Catalog
+
+1. After the semaphore job in the releasing steps is complete, and images have been tagged and pushed, checkout the tag you released.
+
+1. Create the ClusterServiceVersion (CSV), using the tag version for VERSION and the version that this replaces in PREV_VERSION.
+   The versions are semver strings. For example:
+
+   ```
+   make gen-csv VERSION=1.3.1 PREV_VERSION=1.3.0
```

Comment:
The PREV_VERSION will set the `spec.replaces` field in the CSV but in the `make gen-bundle` command we loop through and process all of the CSVs, and may end up overwriting the `spec.replaces` field in some CSVs. This addresses the scenario you've highlighted: adding a new version that is in-between 2 existing versions.

E.g. say we start with `v1.31 <= v1.6.2`. (The `<=` here will mean "replaces").

Then we have `v1.3.2` that we've released and want to publish. After generating the CSV and the bundle, the new bundle we commit to master will have a version chain that looks like:

```
v1.3.1 <= v1.3.2 <= v1.6.2
```

We will have updated the `spec.replaces` field in the `v1.6.2` CSV from `v1.3.1` to `v1.3.2`.

Reply:
One other thing to call out is that the tooling here is proposing a single channel of versions. I don't think we need anything more than this right now. In the future we could have separate channels for alpha or beta versions (like this [operator](https://github.com/IBM/ibm-ingress-nginx-operator/blob/master/deploy/olm-catalog/ibm-ingress-nginx-operator/ibm-ingress-nginx-operator.package.yaml)) or maybe separate channels per release branch (this would allow a user to stay pinned to a minor version of Calico).

Reply:
So I'm understanding correctly you're saying there is some 'magic' in gen-bundle that will change the v1.6.2 spec.replaces field with v1.3.2 when it originally had v1.3.1.
Have you had any chats about the channels? I think I'm onboard with only having the one.

Reply:
> So I'm understanding correctly you're saying there is some 'magic' in gen-bundle that will change the v1.6.2 spec.replaces field with v1.3.2 when it originally had v1.3.1.

Yes, once the script has copied over any CSVs to `deploy/olm-catalog`, it sorts the CSV versions in descending order and overwrites the spec.replaces field in the CSVs (all but the last CSV since being last it replaces nothing).

> Have you had any chats about the channels? I think I'm onboard with only having the one.

Not yet, just this one we're having now.